benchmark/data_generator.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+import pyarrow as pa
+import pyarrow.parquet as pq
+import pandas as pd
+import numpy as np
+from pathlib import Path
+
+import csv
+
+
```

Comment:
not sure if we need to commit this; but won't hurt

Reply:
We should probably move this to its own repo tbh

Reply:
yeah that's a good idea